src/supertux/game_session.hpp

```diff
@@ -88,6 +95,11 @@ class GameSession final : public Screen,
   /** ends the current level */
   void finish(bool win = true);
   void respawn(const std::string& sectorname, const std::string& spawnpointname);
+  void respawn_with_fade(const std::string& sectorname,
+                         const std::string& spawnpointname,
+                         const FadeType fade_type,
+                         const Vector fade_point,
+                         const bool make_invincible = false);
```

Comment:
Does `const bool` mean that make_invincible is always false?

Reply:
I don't think those two relate. It's just that the parameter can't be modified.

Reply:
Indeed, it just says that the parameter cannot be altered inside the function body.